vocabulary/clams.vocabulary.yaml

```diff
@@ -174,6 +188,115 @@ description: >-
   pointing directly into primary data (by using start and end offsets) or
   by linking to one or more other Annotations with the targets property.
 
+properties: 
+  text:
+    type: String
+    description: The surface string in the primary data covered by this span.
+---
+
+name: Token
+parent: Span
+
+similarTo:
+  - http://vocab.lappsgrid.org/Token
```

Comment:
We could add http://www.isocat.org/datcat/DC-1403, which was expressed in the LAPPS vocab

Reply:
Unfortunately, the good-old isocat registry seems to be gone now. (The link redirects to datcatinfo.net website, and returns 404)